_data/simple-icons.json

```diff
@@ -10094,7 +10100,7 @@
         {
             "title": "Windows 95",
             "hex": "008080",
-            "source": "https://en.wikipedia.org/wiki/Windows_95"
+            "source": "https://en.wikipedia.org/wiki/File:Windows_Logo_(1992-2001).svg"
```

Comment:
Could also be the wikimedia link: https://commons.wikimedia.org/wiki/File:Windows_Logo_(1992-2001).svg